apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -929,6 +929,19 @@ type Machine interface {
 	MachineTag() names.MachineTag
 }
 
+// BridgePolicy is an indirection for containerizer.BridgePolicy.
+type BridgePolicy interface {
```

Comment:
Before introducing this indirection, we were mocking expectations for arguments passed to `BridgePolicy` methods. This is obviously leaky as hell - we shouldn't be mocking behaviour expected by a dependency of ours, rather just mocking the dependency.